daisi/scenarios/cpps/default.yml

```diff
@@ -18,8 +18,9 @@ topology:
 
 # required:
 initialNumberNodesAMRs: 12
-numberMaterialFlows: 5
-numberMaterialFlowNodes: 5
```

Comment:
We should not completely change the scenario file every time we introduce a new feature. You can make these changes locally for your testing, but keep the old scenario with the decentralized assignment strategy for consistency